### PR TITLE
fix: avoid cookie banner briefly appearing despite response

### LINKED ISSUE #438: Cookie banner display bugs
**Describe the bug**
- banner flashes up briefly if declined / accepted previously.
- banner is yellow on mobile
- other small styling improvements as appropriate

**Expected behavior**
- init banner as collapsed or wait for state before rendering anything at all
- mobile colour same as desktop

**Screenshots**
Screenshot_20200610_135828_com android chrome

### PR DESCRIPTION
**Pull request by falconx**
Fixes #438 

## Description

SSR version was rendering before the client could run the cookie check.

### CODE REVIEW

File: src/components/CookieConsentBanner.js [hunk 1]
```diff
@@ -38,9 +38,12 @@ const stopGA = () => {
 const CookieConsentBanner = () => {
   const [isBannerOpen, setIsBannerOpen] = React.useState(false)
   const [hasTrackedView, setHasTrackedView] = React.useState(false)
-  const [acceptanceCookie, setAcceptanceCookie] = React.useState(
-    Cookies.get(ACCEPTANCE_COOKIE)
-  )
+  const [acceptanceCookie, setAcceptanceCookie] = React.useState(false)
+
+  // #438: perform check client side to avoid cookie banner briefly appearing
+  React.useEffect(() => {
```
Comment (katjam) [inline src/components/CookieConsentBanner.js:44]: My shallow understanding of `useEffect` is that it gives the component lifecycle... so I am guessing this means that it does the stuff in the function after the component has been through them. If so. Seems reasonable.

File: src/components/CookieConsentBanner.js [hunk 2]
```diff
@@ -145,7 +148,7 @@ const CookieConsentBanner = () => {
           background-color: ${c_COOKIE_BUTTON_HOVER};
         }
         .cookie-banner {
-          background: yellow;
+          background: ${c_COOKIE_BACKGROUND};
```
Comment (katjam) [inline src/components/CookieConsentBanner.js:151]: Nice!

File: src/components/CookieConsentBanner.js (hunk 1 above)
Comment (falconx) [reply src/components/CookieConsentBanner.js:44]: So in this case, the `useEffect` is equivalent to a `componentDidMount`. This lifecycle method does not get called during SSR.

**Comment by falconx** [comment]
Closed for https://github.com/neontribe/www/pull/462

**Pull request closed without merging**